experiments/hierarchical_baselines/src/run_baselines.py

```diff
@@ -0,0 +1,318 @@
+        for model in hrec.level_names.keys():
+            crps_results[model] = []
+            for level in crps_results['Level']:
+                if level=='Overall':
+                    row_idxs = np.arange(len(S_df))
+                else:
+                    row_idxs = S_df.index.get_indexer(tags[level])
+                col_idxs = model_columns.get_indexer(hrec.level_names[model])
+                _y = y_test[row_idxs,:]
+                _y_rec_seeds = y_rec[:,row_idxs,:,:][:,:,:,col_idxs]
+
+                level_model_crps =[]
+                for seed_idx in range(y_rec.shape[0]):
+                    _y_rec = _y_rec_seeds[seed_idx,:,:,:]
+                    level_model_crps.append(scaled_crps(y=_y, y_hat=_y_rec,
+                                                        quantiles=QUANTILES))
+                level_model_crps = f'{np.mean(level_model_crps):.4f}±{(1.96 * np.std(level_model_crps)):.4f}'
+                crps_results[model].append(level_model_crps)
+
+        crps_results = pd.DataFrame(crps_results)
+
+    with CodeTimer('Evaluate Models MSSE  ', verbose):
+        for model in hrec.level_names.keys():
+            msse_results[model] = []
+            for level in msse_results['Level']:
+                if level=='Overall':
+                    row_idxs = np.arange(len(S_df))
+                else:
+                    row_idxs = S_df.index.get_indexer(tags[level])
+                col_idx = model_columns.get_loc(model)
+                _y = y_test[row_idxs,:]
+                _y_train = y_train[row_idxs,:]
+                _y_hat_seeds = y_rec[:,row_idxs,:,:][:,:,:,col_idx]
+
+                level_model_msse = []
+                for seed_idx in range(y_rec.shape[0]):
+                    _y_hat = _y_hat_seeds[seed_idx,:,:]
+                    level_model_msse.append(msse(y=_y, y_hat=_y_hat, y_train=_y_train))
+                level_model_msse = f'{np.mean(level_model_msse):.4f}'
+                msse_results[model].append(level_model_msse)
+
+        msse_results = pd.DataFrame(msse_results)
+
+        return crps_results, msse_results
+
+if __name__ == '__main__':
+    
+    # Parse execution parameters
+    verbose = True
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-intervals_method", "--intervals_method", type=str)
+
+    args = parser.parse_args()
+    intervals_method = args.intervals_method
+
+    assert intervals_method in ['bootstrap', 'normality', 'permbu'], \
+        "Select `intervals_method` from ['bootstrap', 'normality', 'permbu']"
+    
+    print(f'\n {intervals_method.upper()} statistical baselines evaluation \n')
+
+    LEVEL = np.arange(0, 100, 2)
+    qs = [[50-lv/2, 50+lv/2] for lv in LEVEL]
+    QUANTILES = np.sort(np.concatenate(qs)/100)
+
+    # Run experiments
+    crps_results_list = []
+    msse_results_list = []
+    #for dataset in ['Labour', 'Traffic', 'TourismSmall', 'TourismLarge', 'Wiki2']:
+    for dataset in ['OldTraffic', 'OldTourismLarge']:
```

Comment:
Is there a particular reason to exclude the remaining datasets? It might be better to have an argument called `--dataset` that can receive any available dataset.

Reply:
Thanks for the suggestion; I will remember it for the next PR.
This time I saw the opportunity to make a minor improvement while working on the baselines.

I added the `--dataset` script argument.